content/post/his-peer-almighty.ar.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "ترادف خَلقا، تجلى شأنا."
+date = "2022-12-03"
+author = "يوسف خالد"
+cover = "img/his-peer-almighty.jpg"
+CoverCaption = "Jessica Roux"
+images = ['img/his-peer-almighty.jpg']
++++
+`
+مَثَلُك الأعلى ، وموجّهك. قدوتك ، ومعلّمك. أكثرهم علماً، لأجله تفنى. أكملهم صواباً، فوقهم يعلى. بشرٌ كالبشر، ولكنه أسمى.
+
+بيده الأمر كله، أن يرسلك بعيداً، أو يجذبك إليه. يمسك أبواب الجنان، أمّا الجحيم فعلى مصرعيه. بشرٌ كالبشر، ولكنك مكنّته عليك.
+
+يكذب و يسرق. جبالا من الذنوب في عنُقه، فاشهدوا سرّه. يغتصب و يقتل، غيرَ مبالٍ بذنبه. بشرٌ كالبشر، و لازلت بأثره.
+
+الوسيط، لتلك الجنان. لن تعبرها حتى يسمح لك. ليس وفق قانون، ولا حق. فلتؤمن به لذاتِه، و إلا فالنار أحق بك. بَشرٌ كالبشر، وسيقبع معك بقبرك.
```

Comment:
```suggestion
الوسيط، لتلك الجنان. لن تعبرها حتى يسمح لك. ليس وفق قانون، ولا حق. فلتؤمن به لذاتِه، و إلا فالنار بك أحق. بَشرٌ كالبشر، وسيقبع معك بقبرك.
```

Reply:
I thought I did it as you wrote it. Weird.